egs2/how2/sum1/local/path.sh

```diff
@@ -0,0 +1,7 @@
+# check extra module installation
```

Comment:
You can make it a blank file like https://github.com/espnet/espnet/blob/master/egs2/ami/asr1/local/path.sh

Reply:
Or probably, we can check whether we can import longformer library and add the longformer installation instruction here?